### PR TITLE
Implements xmm and fpu internal reg getters

### PR DESCRIPTION
**Pull request by SuchAFuriousDeath**
I'm not sure about u128::from_le_bytes and how to name the getters for the internal fpu registers, the names in the Linux struct don't match the ones in the gdb_stub.

After adding mxcsr, we panic on read_registers.

### CODE REVIEW

File: gui/src/vmm/hv/linux/x86_64.rs [hunk 1]
```diff
@@ -262,6 +262,106 @@ impl<'a> CpuStates for KvmStates<'a> {
...
+        Ok((self.fregs.last_ip & 0xFFFFFFFF) as u32)
+    }
+
+    fn get_foseg(&mut self) -> Result<u32, Self::Err> {
+        Ok((self.fregs.last_dp >> 32) as u32)
+    }
+
+    fn get_fooff(&mut self) -> Result<u32, Self::Err> {
+        Ok((self.fregs.last_dp & 0xFFFFFFFF) as u32)
+    }
```
Comment (ultimaweapon) [inline gui/src/vmm/hv/linux/x86_64.rs:292]: This ones looks weird. Should `seg` part come from the lower words instead of upper words?

Comment (SuchAFuriousDeath) [reply gui/src/vmm/hv/linux/x86_64.rs]: Swapped

Comment (ultimaweapon) [reply gui/src/vmm/hv/linux/x86_64.rs]: I'm not sure if my guess is correct. If you know your original version is a correct one then we should go with it.

**Review by ultimaweapon — APPROVED** [review]

File: gui/src/vmm/hv/linux/x86_64.rs (hunk 1 above)
Comment (SuchAFuriousDeath) [reply gui/src/vmm/hv/linux/x86_64.rs]: image
image

**Pull request merged**